### PR TITLE
Eclipselink controler upgrade to LTS 2.414.1

### PR DESCRIPTION
**Pull request by heurtematte**
Eclipselink jenkins controller fails with OOM. Adding resources only to controller via staticAgentCount=4 allows to manage eclipselink-pull-request-verifier job. 

Upgrade to 2..414.1 jenkins latest lts.

Related to https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3657

### CODE REVIEW

**Comment by heurtematte** [comment]
Finally this PR is only about upgrading to the latest jenkins LTS

File: instances/ee4j.eclipselink/config.jsonnet [hunk 1]
```diff
@@ -6,7 +6,9 @@
     resourcePacks: 5,
   },
   jenkins+: {
+    version: "2.414.1",
     plugins+: [
+      "gerrit-trigger",
```
Comment (fredg02) [inline instances/ee4j.eclipselink/config.jsonnet:11]: Why was `gerrit-trigger` plugin added? Copy & paste error?

Comment (heurtematte) [reply instances/ee4j.eclipselink/config.jsonnet]: plugin deleted!

Comment (fredg02) [reply instances/ee4j.eclipselink/config.jsonnet]: Thank you!

**Review by fredg02 — APPROVED** [review]

**Pull request merged**